Update CONTRIBUTING.md (#7702)
opensearch-project/documentation-website: CONTRIBUTING.md

#### Troubleshooting

> If you encounter one of the following errors when trying to build the documentation website, these troubleshooting tips can help.  

Reviewer: ```suggestion
Try the following troubleshooting steps if you encounter an error when trying to build the documentation website:  
```

> - If you see the `Error running '__rvm_make -j10'`error when running `rvm install 3.2`, you can resolve it by running `rvm install 3.2.0 -C --with-openssl-dir=/opt/homebrew/opt/openssl@3.2` instead of `rvm install 3.2`.

Reviewer: ```suggestion
- If you see the `Error running '__rvm_make -j10'` error when running `rvm install 3.2`, you can resolve it by running `rvm install 3.2.0 -C --with-openssl-dir=/opt/homebrew/opt/openssl@3.2` instead of `rvm install 3.2`.
```
